v2.0/app/components/Forms/helpers/helpers.js

```diff
@@ -1,8 +1,8 @@
 export default function isImage(file) {
   const fileName = file.name || file.path;
-  const suffix = fileName.substr(fileName.indexOf('.') + 1).toLowerCase();
+  const suffix = fileName.split('.').slice(-1)[0].toLowerCase();
```

Comment:
always make sure the length >=1 before you substring with [0]

Reply:
change made here https://github.com/massenergize/frontend-admin/pull/179/commits/0c6c5c6ec7e24ef43cd08823c5f9d3b79393f684